src/core/src/validation_util.cpp

```diff
@@ -1642,3 +1644,13 @@ bool ngraph::validate_host_tensor_vector(const HostTensorVector& tensor_vector,
         return t != nullptr;
     });
 }
+
+void ov::generate_transpose_default_order(std::vector<int64_t>& axes_order, const size_t length) {
+    axes_order.reserve(length);
+    std::generate_n(std::back_inserter(axes_order), length, ov::SeqGen<size_t, ov::Direction::BACKWARD>(length - 1));
```

Comment:
What are the advantages of `std::generate_n` + custom SeqGen instead of `std::iota`?

Reply:
`generate` create elements `iota` fills allocated container.